src/docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -227,17 +227,23 @@ This feature is currently only supported for Ruby, Python, Php, Node, JavaScript
 
 </Note>
 
+<Note>
+Sentry will automatically try to set this up for your organization for Python and JavaScript projects for organizations with the Github integration installed. You are still welcome to add code mappings youself, however, we will still try to do so. Support for other languages and other source code integrations will follow.
+</Note>
+
 Stack trace linking takes you from a file in your Sentry stack trace to that same file in your source code. If you have commit tracking set up in Sentry, we can take you to the exact version (using the commit associated with the event) of the source code when the error occurred. Otherwise we'll link you to the current state of the source code (using the default **branch**).
 
+#### Manual set up
+
 1. Navigate to **Settings > Integrations > GitHub > Configurations**.
 
-1. Click the "Configure" button next to your GitHub Instance.
+2. Click the "Configure" button next to your GitHub Instance.
```

Comment:
Why do some pages just have a `1.`? What's the standard that should be followed?